### PR TITLE
fix(styles): add Horizon delta for User Menu [ci visual]

### PR DESCRIPTION
**Pull request by InnaAtanasova**
## Related Issue
Closes https://github.com/SAP/fundamental-styles/issues/4091

### CODE REVIEW

**Review by mikerodonnell89 — CHANGES_REQUESTED** [review]
I would think when there is a footer present, the last list item should not have bottom border radii when focused:

Screenshot 2023-01-11 at 10 17 46 AM

**Review by platon-rov — COMMENTED** [review]
Why do we have such big spacing?

image

Looks like in wiki it's 8px always
image

Plus, rounding of focus outline seems redundant when there are paddings.

**Comment by InnaAtanasova** [comment]
> https://deploy-preview-4168--fundamental-styles.netlify.app



> I would think when there is a footer present, the last list item should not have bottom border radii when focused:
> 
> Screenshot 2023-01-11 at 10 17 46 AM

This is List component used inside User Menu. The fix should be coming from List, not this component (out of this PR scope)

**Comment by InnaAtanasova** [comment]
> Why do we have such big spacing?
> 
> image
> 
> Looks like in wiki it's 8px always image
> 
> Plus, rounding of focus outline seems redundant when there are paddings.

For the rounded corners: This is List component used inside User Menu. The fix should be coming from List, not this component (out of this PR scope). If we fix it only here, other components that have similar case will also have to fix it.

As for the spacing, the Popover paddings are these:
Screenshot 2023-01-13 at 10 19 01 AM

**Comment by InnaAtanasova** [comment]
created an issue for the rounded corners: https://github.com/SAP/fundamental-styles/issues/4173

**Comment by InnaAtanasova** [comment]
> I would think when there is a footer present, the last list item should not have bottom border radii when focused:
> 
> Screenshot 2023-01-11 at 10 17 46 AM

fixed

**Review by mikerodonnell89 — APPROVED** [review]

File: packages/styles/src/user-menu.scss [hunk 1]
```diff
@@ -25,6 +25,12 @@ $block: #{$fd-namespace}-user-menu;
     &.#{$block}__body--default {
       width: var(--fdUserMenu_Body_Width);
     }
+
+    ul:last-child li:last-child::before,
+    ul li:last-child::before,
+    ul li:last-child a::before {
+      border-radius: 0 !important;
```
Comment (mikerodonnell89) [inline packages/styles/src/user-menu.scss:32]: Probably first-child too?
Screenshot 2023-01-24 at 4 01 24 PM

**Review by mikerodonnell89 — CHANGES_REQUESTED** [review]

File: packages/styles/src/user-menu.scss (hunk 1 above)
Comment (InnaAtanasova) [reply packages/styles/src/user-menu.scss:35]: fixed

**Review by mikerodonnell89 — APPROVED** [review]

**Pull request merged**